src/patcher.js

```diff
@@ -4,10 +4,11 @@
  */
 const awsSDKPatcher = require('./events/aws_sdk.js');
 const httpPatcher = require('./events/http.js');
+const pgPatcher = require('./events/pg.js');
 const config = require('./config.js');
 
 if (!config.config.isEpsagonPatchDisabled) {
-    [awsSDKPatcher, httpPatcher].forEach((patcher) => {
+    [awsSDKPatcher, httpPatcher, pgPatcher].forEach((patcher) => {
```

Comment:
don't you have to add something to the httpPatcher so it will ignore PG's request? or is it not using it?

Reply:
no using http